Assets/Editor/ModelInspector.cs

```diff
@@ -20,26 +20,32 @@ internal static class Styles
             public static GUIStyle preSliderThumb = "preSliderThumb";
         }
 
-        private PreviewRenderUtility m_PreviewUtility;
+        internal struct PreviewSettings
```

Comment:
Probably could make this into a class (struct here won't gain much, given that the members are not just "trivial" types anyway), and then could make it so that various fields have sensible defaults - like whether drawWire is true, zoomFactor, etc etc.

Since e.g. `RenderMeshPreview` is used by `FrameDebugger.cs`, and it would be good if it can still be called in an easy way with all/most settings having sensible defaults. Same for `RenderMeshPreviewSkipCameraAndLighting` that is used from `BillboardAssetInspector.cs`